inc/queryunion.class.php

```diff
@@ -113,4 +113,18 @@ public function getQuery() {
       }
       return $query;
    }
+
+   /**
+    * Get query parameters
+    *
+    * @return array
+    */
+   public function getParameters() {
+      $union_queries = $this->getQueries();
```

Comment:
do we really need to use the public `getQueries` method instead direct access of `$this->queries` ?

Reply:
Depends if we want verbatim data or not, since `getQueries` can do extra stuff (this is not currently the case).